### ISSUE TITLE
[Feature Request] Customizable location in jplhorizons module

### ISSUE THREAD

**Issue by ysBach** [astropy/astroquery#1180]
If I understand the [latest doc](https://astroquery.readthedocs.io/en/latest/jplhorizons/jplhorizons.html) correctly, it is not possible to use custom `location` (e.g., by specifying longitude/latitude/height). 

For many amateur observations and school's observatories, it is almost essential to customize it.

**Comment by bsipocz** [comment]
cc @mkelley

**Comment by mkelley** [comment]
You are correct and I think this is possible to fix.  My plan for Minor Planet Center queries (MPES) is to allow `(longitude, latitude, altitude)` and astropy's `EarthLocation` as inputs, in addition to observatory code.  However, Horizons is more powerful than the MPES and can have observers placed on sites throughout the solar system, so we should be sure to allow that here.

From the instructions at ftp://ssd.jpl.nasa.gov/pub/ssd/horizons_batch_example.long
```
! COORD_TYPE selects type of user coordinates in SITE_COORD.
! Used only when CENTER = 'coord'. 
! Values: GEODETIC or CYLINDRICAL

 COORD_TYPE = 'GEODETIC'

! SITE_COORD sets coordinates of type COORD_TYPE. If 
! COORD_TYPE=GEODETIC, then SITE_COORD is assumed to be the 
! set {E. long., lat, height} (KM and DEG). 
!
! If COORD_TYPE=CYCLINDRICAL, SITE_COORD is assumed to be 
! {E.lon,Dxy,Dz} also in KM and DEG. Quotes ALWAYS 
! necessary here! See main doc for more details.

 SITE_COORD = '0,0,0'
```

**Comment by mommermi** [comment]
I will implement this with the next update.

**Comment by bsipocz** [comment]
Thanks @mommermi!

@ysBach - You can try this out if you upgrade to the latest release with `pip install --pre --upgrade astroquery`.

**Comment by mommermi** [comment]
this is implemented with #1207 and can be closed now
